### PR TITLE
Connect redux react

### PR DESCRIPTION
**Pull request by Ol-create**
- Generate and add unique id to book
- Dispatch remove book function
- Dispatch checkStatus action
- UseSelector to compare state
- Add dispatch addBook function
- Add styles to button
- Add progress text align to center.

### CODE REVIEW

File: src/components/Books.jsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import BookElement from './SingleBook';
 import AddBook from './AddBook';
```
Comment (rloterh) [inline src/components/Books.jsx]: - [ ] `store` should be imported and the main React component `<App>` should be wrapped in _Provider_ and assign `store` in your _index.js_, similar to:

```ruby
    import store from './redux/configureStore';

    ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
    )
```

Comment (rloterh) [inline src/components/Books.jsx]: Buttons for adding and removing possible, as well as checking status are missing! 
- [ ] Kindly ensure that books can be added and removed, and buttons are visible.
- [ ] Also, ensure that the button for checking the status on the categories page is present and working as expected.

**Your Home page:**
image

**Your categories page:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Oluola, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Highlights 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @Ol-create 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**